app/views/mobile-client-sdk-config/sdk-config-android.html

```diff
@@ -0,0 +1,196 @@
+<article>
+  <h2>Getting Started with AeroGear Services Android SDK</h2>
+  <p>Following guide provides general information for starting with AeroGear Services SDK.</p>
+  <h2 id="user-content-prerequisites">Prerequisites</h2>
+  <p>You will need the following installed on your system</p>
+  <ul>
+    <li>
+      <p>Android SDK or Android Studio from <a href="https://developer.android.com/studio/index.html" rel="nofollow">Google</a>
+      </p>
+    </li>
+  </ul>
+  <p>It is reccomended you have the following installed</p>
+
+  <ul>
+    <li>
+      <p>The latest version of <a href="http://openjdk.java.net/install/index.html" rel="nofollow">OpenJDK</a></p>
+    </li>
+  </ul>
+
+  <h2>Installing</h2>
+  <p>As of the writing of this guide, the Aerogear Android SDK is not in jcenter and needs to be installed manually.</p>
+  <h3>Clone and build the library</h3>
+
+  <pre>
+    git clone https://github.com/aerogear/aerogear-android-sdk
+    cd aerogear-android-sdk
+    ./gradlew install</pre>
+
+  <p>You can verify the install went well by checking your local maven repository</p>
+
+  <pre>
+    ls ~/.m2/repository/org/aerogear
+    # Should output auth core</pre>
+
+
+  <h2>Using the SDK</h2>
+
+  <h3>MavenLocal()</h3>
+
+  <p>If you are using a build of the SDK from source you will need to add mavenLocal() to the build.gradle file in your project’s root directory.</p>
+
+  <pre>
+  allprojects {
+      repositories {
+          mavenLocal() //<-- Add This line
+          google()
+          jcenter()
+      }
+  }</pre>
+
+
+  <h3>Reference the Dependencies</h3>
+
+  <p>In the build.gradle file for your app you can begin adding dependencies provided by the sdk.</p>
+
+
+  <pre>
+  dependencies {
+      ...
+      implementation 'org.aerogear:core:5.0.0-SNAPSHOT'
+      implementation 'org.aerogear:auth:5.0.0-SNAPSHOT' //If you are using IDM services provided by KeyCloak
+  }</pre>
+
+
+  <h3>Using the SDK</h3>
+
+  <p>In an example you can use the SDK as follows</p>
+
+
+  <pre>
+  public class MyApplication extends Application {
```

Comment:
Is there a code prettifier library available?

Reply:
there isnt any being used at the moment, I didnt want to add a new dependency.